test/k8s-integration/config/fs-sc-basic-hdd.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-filestore
+provisioner: filestore.csi.storage.gke.io
+parameters:
+  network: e2e-test-saikatroyc # Change this network as per the GCE deployment
```

Comment:
should we set this to default as an example?

Reply:
Ack